joplin/base/models.py

```diff
@@ -482,6 +481,52 @@ class DepartmentPageDirector(Orderable):
         FieldPanel('about'),
     ]
 
+class OfficialDocumentPage(JanisPage):
```

Comment:
I know the current code base dosen't have this, but I'd love to have us get in the habit of using docstrings
Off the top of  my head: 
"""
This is a page for listing related documents for a department(s?) and arranging them by date.
It's currently essentially a list with a brief summary of each item and an link to the url of the document. 
Eventually this might migrate to/be replaced by a model using Wagtail Documents
"""

Reply:
A fine idea. I will do this.